qtconsole/usage.py

```diff
@@ -0,0 +1,221 @@
+"""
+Usage information for QtConsole
+"""
+
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+
+def page_guiref(arg_s=None):
+    """Show a basic reference about QtConsole."""
+    from IPython.core import page
+    page.page(gui_reference)
+
+
+gui_reference = """\
+=====================
+The Jupyter QtConsole
+=====================
+
+This console is designed to emulate the look, feel and workflow of a terminal
+environment, while adding a number of enhancements that are simply not possible
+in a real terminal, such as inline syntax highlighting, true multiline editing,
```

Comment:
The console is designed to emulate the look, feel and workflow of a terminal environment. Beyond this basic design, the console may also implement functionality not currently found in most terminal emulators. Some examples of these console enhancements are inline syntax highlighting, multiline editing, inline graphics, and others.